src/rules/typedefWhitespaceRule.ts

```diff
@@ -119,4 +125,15 @@ class TypedefWhitespaceWalker extends Lint.RuleWalker {
         const options = allOptions[0];
         return options[option];
     }
+
+    private positionBeforeColon(node: ts.FunctionDeclaration | ts.FunctionExpression | ts.AccessorDeclaration |
```

Comment:
Since this `positionBeforeColon` method never accesses `this.` anything, you can declare it outside of the class. Then above, you can simplify things a little:

```
this.checkSpace("call-signature", node, node.type, positionBeforeColon(node))
```